test/controllers/reports_controller_test.rb

```diff
@@ -0,0 +1,23 @@
+require "test_helper"
+
+class ReportsControllerTest < ActionController::TestCase
+  test "should get index" do
+    get :index
+    assert_response :success
+  end
+
+  test "should get show" do
+    get :show
+    assert_response :success
+  end
+
+  test "should get new" do
+    get :new
+    assert_response :success
+  end
+
+  test "should get edit" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.